packages/app/src/app/pages/Sandbox/Editor/Header/Actions.tsx

```diff
@@ -98,29 +101,35 @@ export const Actions = () => {
         </TooltipButton>
       )}
 
-      {hasLogIn ? (
-        <TooltipButton
-          tooltip={userLiked ? 'Undo like sandbox' : 'Like sandbox'}
-          variant="link"
-          onClick={() => likeSandboxToggled(id)}
-        >
-          <LikeIcon
-            css={css({
-              height: 3,
-              marginRight: 1,
-              color: userLiked ? 'reds.500' : 'inherit',
-            })}
-          />{' '}
-          <span>{likeCount}</span>
-        </TooltipButton>
+      {featureFlags.ACCESS_SHEET && isLive ? (
+        <CollaboratorHeads />
       ) : (
-        <Stack gap={1} paddingX={2} align="center">
-          <LikeIcon css={css({ height: 3 })} />
-          <span>{likeCount}</span>
-        </Stack>
+        <>
+          {hasLogIn ? (
+            <TooltipButton
+              tooltip={userLiked ? 'Undo like sandbox' : 'Like sandbox'}
+              variant="link"
+              onClick={() => likeSandboxToggled(id)}
+            >
+              <LikeIcon
+                css={css({
+                  height: 3,
+                  marginRight: 1,
+                  color: userLiked ? 'reds.500' : 'inherit',
+                })}
+              />{' '}
+              <span>{likeCount}</span>
+            </TooltipButton>
+          ) : (
+            <Stack gap={1} paddingX={2} align="center">
+              <LikeIcon css={css({ height: 3 })} />
+              <span>{likeCount}</span>
+            </Stack>
+          )}
+        </>
       )}
 
-      {user?.curatorAt && (
+      {false && user?.curatorAt && (
```

Comment:
Why not just remove this whole block instead? 🤔

Reply:
Planning to enable it once we've done more testing.